pex/pex_builder.py

```diff
@@ -591,9 +591,11 @@ def _prepare_bootstrap(self):
                 dirs[:] = bootstrap_packages
 
             for f in filter_pyc_files(files):
-                if f.endswith("testing.py"):
```

Comment:
Why this change?

Reply:
Ah, right. You reviewed that one actually in #2197. I noticed this miss in that PR while I was in this code.